### ISSUE TITLE
Segments relative links with dots

### ISSUE THREAD

**Issue by kolchan11** [Novage/p2p-media-loader#18]
Hello,

Thanks for your library!

But I have segment links that look like this:
```
#EXTINF:20.000,
./big_buck_bunny.mp4:hls:seg-1-v1-a1.ts
#EXTINF:20.000,
./big_buck_bunny.mp4:hls:seg-2-v1-a1.ts
#EXTINF:20.000,
./big_buck_bunny.mp4:hls:seg-3-v1-a1.ts
```

And it seems that the library can't handle these "./" at the start of each link, and as a result, p2p does not work.
(`engine.loader.segments` is empty while `Playlist.segments` is not)

**Comment by mrlika** [comment]
Thanks. We know about the problem. The fix is going to be released soon.

**Comment by mrlika** [comment]
Fixed in 94f637d7fe8b96ab8afb9075c2bb32fc182a62d9

**Comment by kolchan11** [comment]
Thank you!

Everything works great now
